app/views/messages/_messages_for_library.html.erb

```diff
@@ -1,24 +1,45 @@
 <div class="messages">
-<% messages = @messages.select { |m| m.library == library_code && m.request_type == request_type } %>
-<% messages.each do |message| %>
-  <div class="message">
+
+  <% messages = @messages.select { |m| m.library == library_code && m.request_type == request_type } %>
+  <% @library = LibraryLocation.library_name_by_code(library_code).to_s %>
+
+  <% if @library.empty? %>
+    <% @library = 'anywhere' %>
+  <% end %>
+
+  <% if messages.none? %>
+    <h2><%= request_type.titleize + " from " + @library %></h2>
```

Comment:
The best I can tell, this is the only place where the `@library` instance variable is used (I don't think this should be an instance variable w/ the `@`).  Can we just change this to be `<%= request_type.titleize + " from #{library || 'anywhere'}" %>`.  Then we could remove lines 6-8 entirely.